src/main/resources/swagger.api/schemas/dto/tenant.json

```diff
@@ -9,6 +9,9 @@
       },
       "name": {
         "type": "string"
+      },
+      "consortiumId": {
```

Comment:
We don't need consortiumId in request/response payloads, it will be presented only in url path

Reply:
But we are suppose to store consortiumId (the reference of Consotium Table). for that we need setter method.

Reply:
thinking the DTO wouldn't have consortiumId but the entity would.